vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -477,3 +478,342 @@ def fused_moe(
                          out=hidden_states)
     return torch.sum(intermediate_cache3.view(*intermediate_cache3.shape),
                      dim=1)
+
```

Comment:
Per my comment below, we will load the already compressed weights via `create_weights`. 

So none of this will need to be called on the hotpath

As a result, all of this should be moved into testing utilities